src/lib/y2storage/boot_requirements_strategies/uefi.rb

```diff
@@ -64,6 +64,17 @@ def warnings
         res + res_new
       end
 
+      private
+
+      # @return [VolumeSpecification]
```

Comment:
Why was did moved from `protected` into `private`? The reason is not obvious to me.

Reply:
bls.rb is derived from uefi.rb and has its own efi_volume function which should be used. Is this wrong ?

Reply:
```ruby
class Foo
  def initialize(name)
    @name = name
  end

  def speak
    puts "I'm #{decorated_name}"
  end

  protected

  def decorated_name
    "**#{@name}**"
  end
end

class Bar < Foo
  protected

  def decorated_name
    "--#{@name}--"
  end
end

foo = Foo.new("John")
foo.speak

bar = Bar.new("John")
bar.speak
```

Reply:
Yes, you are right. Thanks for the lesson. ;-)  Fixed.